.github/workflows/pr.yml

```
@@ -98,7 +255,19 @@
         with:
           issue-number: ${{ github.event.pull_request.number }}
           body: |
-            Your deployment is here ${{needs.setup.outputs.deploy-url}}
+            ## 📚 Versioned Documentation Deployed
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)